Engines/Wine/Plugins/DOS support/script.js

```diff
@@ -135,4 +135,6 @@ Wine.prototype.dosbox = function () {
             return this;
         }
     }
+
+    return this;
```

Comment:
Didn't we have this already?

Reply:
Sometimes, git behaves weird

Reply:
Not sure what you did but I think you just have to merge master to this branch.

Reply:
My branch looks exactly like the other ones. This is really strange

Reply:
I don't know. If you look at the commits in this PR, it already doesn't look right. Normally you should see only what you did in this PR.